cheatSheet.md

```diff
@@ -0,0 +1,48 @@
+```shell
```

Comment:
Can we call this file `tutorial.md` or `getting_started.md` and add a small description at the beginning as if it was a "getting started tutorial"?